bee-message/src/output/mod.rs

```diff
@@ -229,3 +281,12 @@ fn minimum_storage_deposit(config: &ByteCostConfig, address: &Address) -> u64 {
         .unwrap();
     Output::Basic(basic_output).byte_cost(config)
 }
+
+///
+pub fn inputs_commitment<'a>(inputs: impl Iterator<Item = &'a Output>) -> [u8; 32] {
```

Comment:
docs

Reply:
`create_inputs_commitment` ?

Also maybe the corresponding docs could explain that the method generates a Blacke2b256 hash over the inputs so one understands as the consumer of this API what the returned bytes are.

Reply:
Honestly I don't really like that kind of prefixes, it's very much inconsistent as the boundary when to put it or not is not obvious at all. If we follow the logic, anything that returns ownership create something so we should put it in a LOT of places in the repository.

Reply:
I'm not convinced as I still see it as being very much inconsistent but I'll do it.